src/TouristInfoCards.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+const TouristInfoCards = props => {
+  return (
+    <div className="tourist-information">
+      <div className="container">
+        <div className="row">
+          {props.data.map((singleCard, index) => {
```

Comment:
For me this is crying out for a `TouristInfoCard` component.  You could define it within this same file.

Reply:
Good work driving this by looping over data.